src/components/Number.js

```diff
@@ -1,15 +1,26 @@
-const Number = ({ value, onClick }) => {
-  /** TODO: What happens when a user clicks a number, what do we want to pass to our parent? */
+import { useState } from "react";
+import styles from "../style/Button.module.css";
+
+const Number = ({ value, onClick, style}) => {
+  const [clicked, setClicked] = useState(false)
   return (
-    <div
+    <button
       style={{
-        padding: 10,
+        width: "60px",
```

Comment:
No need to include units here. If you just pass a number then it will default to pixels. 

```suggestion
        width: 60,
```